src/store/BarrierStore.ts

```diff
@@ -64,7 +65,7 @@ export default class BarrierStore {
         return this.mainStore.chart.yAxiswidth;
     }
     get priceLabelWidth(): number {
-        return this.yAxisWidth + 1;
+        return 16;
```

Comment:
why are we hardcoding the width? It may not work properly for forex yaxis values as it might have more than 5 decimal places.

Reply:
This number is only relevant for the case when barrier overlaps with current price. The value with all the decimals is moved out of this container and not affected by this width.

Reply:
Can we give a meaningful name? 😅

Reply:
Sure, I replaced the name to something more relevant now.